docs/references/rest-apis/rest-cloudconnection-api.md

```diff
@@ -0,0 +1,726 @@
+        "org.eclipse.kura.data.DataService"
+    ]
+}
+```
+- 500 Internal Server Error
+
+#### Create Cloud Endpoint
+
+- Description: This method create a new [CloudEndpoint](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/CloudEndpoint.html), and the related stack components: [CloudService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloud/CloudService.html), [DataService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/data/DataService.html), [DataTransportService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/data/DataTransportService.html)
+- Method: POST
+- API PATH: `services/cloudconnection/v1/cloudEndpoint`
+
+##### Request
+```JSON
+{
+    "cloudConnectionFactoryPid" : "org.eclipse.kura.cloud.CloudService",
+    "cloudEndpointPid" : "org.eclipse.kura.cloud.CloudService-1"
+}
+```
+
+##### Responses
+
+- 204 Ok Status
+- 400 malformed `cloudConnectionFactoryPid` or `cloudEndpointPid`.
+- 404 Wrong `cloudConnectionFactoryPid`
+- 500 Internal Server Error
+
+#### Create Publisher/Subscriber instance
+
+- Description: his method create a new [CloudPublisher](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/publisher/CloudPublisher.html) or a [CloudSubscriber](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/subscriber/CloudSubscriber.html) instance for a specific CloudEndpoint [CloudEndpoint](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/CloudEndpoint.html). The type of the instance depends from the type of the specified factory. 
+- Method: POST
+- API PATH: `services/cloudconnection/v1/pubSub`
+
+##### Request
+```JSON
+{
+    "pid" : "testPub",
+    "factoryPid" : "org.eclipse.kura.cloud.publisher.CloudPublisher",
+    "cloudEndpointPid" : "org.eclipse.kura.cloud.CloudService"
+}
+```
+
+##### Responses
+
+- 204 Ok Status
+- 400 malformed `pid`, `factoryPid` or `cloudEndpointPid` 
+- 404 If `factoryPid` or `cloudEndpointPid` are wrong.
+
+#### Get Configurations
+
+- Description: This method retrieves the complete configuration, including the metatype description, of the component instance which are [CloudPublisher](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/publisher/CloudPublisher.html) or a [CloudSubscriber](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/subscriber/CloudSubscriber.html) or [CloudService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloud/CloudService.html), [DataService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/data/DataService.html), [DataTransportService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/data/DataTransportService.html)
```

Comment:
Maybe better to say that it returns the configuration of publishers, subscribers and cloud stack components whose pid is returend by the get stack component pids call.